.packit.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     # x86_64 is assumed by default
     # qm is noarch so we only need to test on one arch
     targets:
-      fedora-rawhide: {}
+      fedora-development: {}
```

Comment:
Configuring only `fedora-rawhide` and `fedora-latest` means the job will run for `rawhide` and `f40`, but not `f41`. But if that's what you want, I can revert this change.

Reply:
No, that's fine.